internal/driver/glfw/loop.go

```diff
@@ -130,6 +130,15 @@ func (d *gLDriver) runGL() {
 			if f := l.OnStopped(); f != nil {
 				l.QueueEvent(f)
```

Comment:
Does QueueEvent add f to the funcQueue? If so, will the added code below inadvertently make it so that the OnStopped callback is never invoked now? We should just be safe to call OnStopped directly here, since we're on main?

Reply:
I'm not sure I follow, but with my added comment it should illustrate that things are still getting called because we drain that function queue.
It may be safe to call directly, but I think that is a different tidy where lifecycle is updated for the new threading.